lib/active_merchant/billing/gateways/secure_trading.rb

```diff
@@ -0,0 +1,274 @@
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    # For more information on the Secure Trading visit
+    # {SecureTraining}[https://docs.securetrading.com]
+    #
+    class SecureTradingGateway < Gateway
+      self.live_url = 'https://webservices.securetrading.net:443/xml/'
+
+      self.supported_countries = ['US']
```

Comment:
I think this needs to be expanded because I believe our first customer will be UK based (and SecureTrading is, as far as I remember a UK based company).

Reply:
Should we use `GB` as country? 

https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/country.rb#L302

Reply:
I've pushed a commit expanding countries to `GB`